rnndb/graph/g80_compute.xml

```diff
@@ -331,11 +331,11 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
 				in case of 2D surface</doc>
 			</reg32>
 			<reg32 offset="0x10" name="MODE">
-				<bitfield name="LINEAR" pos="0" type="boolean"/>
-				<bitfield name="UNK1" high="7" low="4">
+				<bitfield name="FORMAT" pos="0" type="inv_memory_format"/>
```

Comment:
Don't use the generic type here - the choices are really LINEAR and BLOCKLINEAR - the linear choice disables 2D addressing, so it shouldn't be called PITCH.